fermenter-mda/src/test/java/org/technologybrewery/fermenter/mda/TestUtils.java

```diff
@@ -0,0 +1,32 @@
+package org.technologybrewery.fermenter.mda;
+
+import io.cucumber.java.ParameterType;
+import org.codehaus.plexus.util.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class TestUtils {
+
+    @ParameterType(value = "[^\"]*")
+    public Boolean booleanValue(String value) {
+        return Boolean.valueOf(value);
+    }
+
+    @ParameterType(value = "[^\"]*")
```

Comment:
I (Informational): covered how this works in the following blog post: https://codifyiq.substack.com/p/stupid-cucumber-tricks?r=2f1wz9